### ISSUE TITLE
[shards install] shard name () doesn't match dependency name (financialmodelingprep)

### ISSUE THREAD

**Issue by grokify** [grokify/crystal-financialmodelingprep#3]
From a dependent:

```bash
% shards install
Resolving dependencies
Fetching https://github.com/grokify/crystal-financialmodelingprep.git
Fetching https://github.com/mamantoha/crest.git
Fetching https://github.com/mamantoha/http-client-digest_auth.git
Fetching https://github.com/mamantoha/http_proxy.git
Error shard name () doesn't match dependency name (financialmodelingprep)
```

The following is from the dependent `shard.yml` file. No version is specified.

```yaml
dependencies:
  financialmodelingprep:
    github: grokify/crystal-financialmodelingprep
```

Unknown why the name is empty in the error message: `()`.

References:

1. `crystal-financialmodelingprep` was generated with `shardName=financialmodelingprep`
1. `crystal-financialmodelingprep`'s `shard.yml` file has `name: financialmodelingprep`

**Comment by grokify** [comment]
Perhaps issue is that module name is `FinancialModelingPrep` and needs to be `Financialmodelingprep` as `crystal-pingpong` works:

https://github.com/grokify/crystal-pingpong

**Comment by grokify** [comment]
Resolved in 38f31b5345f3e3f22f284c54283cf05d9a2e9e29 by changing module name to `Financialmodelingprep`.
